cli/cmd/team_members.go

```diff
@@ -0,0 +1,392 @@
+		Args:  cobra.ExactArgs(1),
+		RunE: func(_ *cobra.Command, args []string) error {
+			cli.StartProgress(" Deleting team member...")
+			err := cli.LwApi.V2.TeamMembers.Delete(args[0])
+			cli.StopProgress()
+			if err != nil {
+				return errors.Wrap(err, "unable to delete team member")
+			}
+			cli.OutputHuman("The team member with GUID %s was deleted\n", args[0])
+			return nil
+		},
+	}
+
+	// create command is used to create a new lacework team member
+	teamMembersCreateCommand = &cobra.Command{
+		Use:   "create",
+		Short: "Create a new team member",
+		Args:  cobra.NoArgs,
+		RunE: func(_ *cobra.Command, args []string) error {
+			if !cli.InteractiveMode() {
+				return errors.New("interactive mode is disabled")
+			}
+
+			userID, err := promptCreateTeamMember()
+			if err != nil {
+				return errors.Wrap(err, "unable to create team member")
+			}
+
+			cli.OutputHuman("The team member was created with GUID %s\n", userID)
+			return nil
+		},
+	}
+)
+
+func init() {
+	// add the team-member command
+	rootCmd.AddCommand(teamMembersCommand)
+
+	// add sub-commands to the team-member command
+	teamMembersCommand.AddCommand(teamMembersListCommand)
+	teamMembersCommand.AddCommand(teamMembersShowCommand)
+	teamMembersCommand.AddCommand(teamMembersCreateCommand)
+	teamMembersCommand.AddCommand(teamMembersDeleteCommand)
+}
+
+func buildTeamMemberDetailsTable(tm api.TeamMember) string {
+	var (
+		details     [][]string
+		updatedTime string
+	)
+
+	if tm.Props.UpdatedTime != nil {
+		updatedTime = fmt.Sprintf("%v", tm.Props.UpdatedTime)
+	}
+
+	details = append(details, []string{"FIRST NAME", tm.Props.FirstName})
+	details = append(details, []string{"LAST NAME", tm.Props.LastName})
+	details = append(details, []string{"COMPANY", tm.Props.Company})
+	details = append(details, []string{"ACCOUNT ADMIN", strconv.FormatBool(tm.Props.AccountAdmin)})
+	details = append(details, []string{"ORG ADMIN", strconv.FormatBool(tm.Props.OrgAdmin)})
+	details = append(details, []string{"CREATED AT", tm.Props.CreatedTime})
+	details = append(details, []string{"JIT CREATED", strconv.FormatBool(tm.Props.JitCreated)})
+	details = append(details, []string{"UPDATED BY", tm.Props.UpdatedBy})
+	details = append(details, []string{"UPDATED AT", updatedTime})
+
+	detailsTable := &strings.Builder{}
+	detailsTable.WriteString(renderOneLineCustomTable("TEAM MEMBER DETAILS",
+		renderCustomTable([]string{}, details,
+			tableFunc(func(t *tablewriter.Table) {
+				t.SetBorder(false)
+				t.SetColumnSeparator(" ")
+				t.SetAutoWrapText(false)
+				t.SetAlignment(tablewriter.ALIGN_LEFT)
+			}),
+		),
+		tableFunc(func(t *tablewriter.Table) {
+			t.SetBorder(false)
+			t.SetAutoWrapText(false)
+		}),
+	),
+	)
+
+	detailsTable.WriteString("\n")
+	return detailsTable.String()
+}
+
+func promptCreateTeamMember() (string, error) {
+	questions := []*survey.Question{
+		{
+			Name:     "email",
+			Prompt:   &survey.Input{Message: "Email: "},
+			Validate: survey.Required,
```

Comment:
Can we add a simple validation to this field, at least check that the string has an `@` and a `.`

Reply:
That way we don't send the request to the APIs to have it return
```
ERROR unable to create team member:
  [POST] https://customerdemo.lacework.net/api/v2/TeamMembers
  [400] validemail: invalid value for userName; username: not a well-formed email address;
```